report_qweb_signer/models/ir_actions_report.py

```diff
@@ -195,28 +195,32 @@ def pdf_sign(self, pdf, certificate):
             if process.returncode:
                 raise UserError(
                     _(
-                        "Signing report (PDF): jPdfSign failed (error code: %s). "
-                        "Message: %s. Output: %s"
+                        "Signing report (PDF): jPdfSign failed (error code: {error_code}). "
+                        "Message: {message}. Output: {output}"
+                    ).format(
+                        error_code=process.returncode,
```

Comment:
I think OCA guidelines are to avoid string formatting with format as it has more security risks. Note that you can keep your change by simply replacing the changes by `%(message)s`, see https://github.com/odoo/odoo/commit/3c3b88f6768e31417263b3828a874ca204e29514.

Reply:
Yes, this is required, good spot! Newer guidelines also require to use named placeholders when multiple placeholders appear in the same translated string, so just follow @len-foss' suggestion and you'll be good to go.

Reply:
sorry for the delay, it should be fixed now